README.md

```diff
@@ -41,26 +98,37 @@ If it modifies the CLI, provide the env variable name, the argument name, and th
 
 This section serves as a user-level guide and should resemble official documentation. Anything that the user may encounter while interacting with the feature should be presented here.
 
-### V. Impact on Documentation
+#### V. Impact on Documentation
 
 If the feature requires additions or updates to the documentation, they should be noted here. It's the role of the documentation team to ensure this section of the feature specification is accurate.
 
-## 2. Technical Specifications
+#### VI. Impact on SDKs
+
+If the feature requires additions or updates to the SDks, they should be noted here. It's the role of the integration team to ensure this section of the feature specification is accurate.
+
+### 2. Technical Specifications
 
-This section of the feature specification document has a much narrower audience: the developer(s) that will implement the feature. Its goal is to discuss and clarify how the feature will be developed and implemented. It has three subsections.
+This section of the feature specification document has a much narrower audience: the developer(s) that will implement the feature. Its goal is to discuss and clarify how the feature will be developed and implemented. It has four subsections.
 
-### I. Architecture
+Architecture and Implementation Details parts should be considered as critical when the specification is owned by someone outside MeiliSearch teams. Internal technical teams may use this part to keep track of technical brainstorms, related proto code or basecode examples.
+
+#### I. Architecture
 
 This section presents how the new feature will fit into the existing architecture and codebase on an abstract level. It should also define a scope where the feature exists.
 
-### II. Implementation Details
+#### II. Implementation Details
 
 This section is for precise, practical aspects of implementation, e.g. interfaces, potential code problems, or specific algorithmic choices.
 
-### III. Corner Cases
+#### III. Issues Summary
+
+This section is for listing any related issues that need to be done to fully implement this specification.
```

Comment:
I think we should detail:
- the labels `Spec Releated` has to be added to the issues
- the issues have to mention the spec in the description
- the issues must absolutely link to the PRs that resolve the spec, so that we can easily track them